web/themes/joinup/templates/layout/page.html.twig

```diff
@@ -77,7 +77,7 @@
         </aside>
       {% endif %}
 
-      <div class="section section--main mdl-cell {{ has_vertical_tabs ? 'mdl-cell--12-col mdl-grid--no-spacing content-container--form' : 'mdl-cell--8-col content-container' }}">
+      <div class="section section--main {{ container_class }} mdl-cell {{ has_vertical_tabs ? 'mdl-cell--12-col mdl-grid--no-spacing' : 'mdl-cell--8-col' }}">
```

Comment:
Not a real problem, but could you please order static and dynamic classes? I mean, container_class is in the middle of static classes and it's a bit unreadable.